packages/loader/container-loader/src/container.ts

```diff
@@ -1905,8 +1910,8 @@ export class Container
 	) {
 		// Log actual event
 		const time = performance.now();
-		this.connectionTransitionTimes[value] = time;
 		const duration = time - this.connectionTransitionTimes[oldState];
+		this.connectionTransitionTimes[value] = time;
```

Comment:
Is this change noop? It should, unless oldState == value. If it could happen, telemetry would look weird, so I think it's worth understanding it better